Add wiko packages

jvirgovic opened the pull request:
closes #856.

Bqleine committed:
Add information for com.miui.phrase

AnonymousWP committed:
Mention potential breakage of app in description

This commit also contains the change of `removal` from `Recommended` to `Advanced`

Bqleine committed:
Update resources/assets/uad_lists.json

Co-authored-by: AnonymousWP <50231698+AnonymousWP@users.noreply.github.com>

AnonymousWP committed:
Add fonts

AnonymousWP committed:
Add Realme packages

AnonymousWP committed:
Add pending Realme packages

AnonymousWP committed:
Add Infinix/Tecno packages

AnonymousWP committed:
Add Huawei/Honor packages

AnonymousWP committed:
Add `com.kuackmedia.orange`

AnonymousWP committed:
Add Honor and Huawei packages

AnonymousWP committed:
Add leftover Huawei and Honor packages

AnonymousWP committed:
Add description to `com.samsung.android.appseparation`

Fixes #524.

AnonymousWP committed:
Change `removal` from Expert to Advanced

AnonymousWP committed:
Add description to `com.samsung.android.scs`

This commit also changes the `list` from 'Pending' to 'Oem'

AnonymousWP committed:
Add packages and change removals

AnonymousWP committed:
Fix Samsung app descriptions

AnonymousWP committed:
Update description of `com.android.stk` and `com.android.stk2`

AnonymousWP committed:
Update description of `com.samsung.android.mobileservice`

AnonymousWP committed:
Add dependency note to description of `com.android.stk` packages

AnonymousWP committed:
Add `com.huawei.himovie`

AnonymousWP committed:
Add `com.sec.android.autodoodle.service`

AnonymousWP committed:
Modify description of `com.samsung.android.securitylogagent`

AnonymousWP committed:
Add links to description of `com.samsung.android.securitylogagent`

AnonymousWP committed:
Add `com.sec.app.samsungprintservice`

AnonymousWP committed:
Edit descriptions regarding feedback

AnonymousWP committed:
Add Xiaomi packages

AnonymousWP committed:
Add Motorola packages

AnonymousWP committed:
Change description of `com.android.bookmarkprovider`

AnonymousWP committed:
Change description of `com.google.android.webview`

AnonymousWP committed:
description: add use of Android WebView

AnonymousWP committed:
change(description): add Google Discover to `com.google.android.googlequicksearchbox`

AnonymousWP committed:
pkg: add LDPlayer packages

AnonymousWP committed:
pkg: specify default browser LDPlayer

AnonymousWP committed:
pkg: specify default gallery LDPlayer

AnonymousWP committed:
pkg: add `com.android.basicsmsreceiver`

AnonymousWP committed:
pkg: add Fairphone apps

AnonymousWP committed:
Fix incorrect package ID

AnonymousWP committed:
pkg: add `com.android.theme.font.notoserifsource`

AnonymousWP committed:
pkg(description): add notes to Samsung biometric services apps

AnonymousWP committed:
Add three Huawei packages

`com.huawei.hwdockbar`, `com.huawei.filemanager` and `com.huawei.hms5gkit.agentservice`

AnonymousWP committed:
pkg: add Fairphone packages

AnonymousWP committed:
pkg: update description and removal of `com.miui.securitycenter`

AnonymousWP committed:
pkg: change description and removal of `com.miui.powerkeeper`

AnonymousWP committed:
pkg: add `com.samsung.android.knox.kpecore`

Fixes #610

AnonymousWP committed:
pkg: add alt name for `com.samsung.android.app.omcagent`

AnonymousWP committed:
pkg: add Sony packages

Fixes #584.

AnonymousWP committed:
pkg: change packages to advanced

AnonymousWP committed:
pkg: add Yandex apps

Fixes #617.

AnonymousWP committed:
pkg: add `com.mi.globalminusscreen`

Fixes #619

AnonymousWP committed:
pkg: add Honor Honor 10 Lite packages

Fixes #624

AnonymousWP committed:
pkg: add more Huawei packages

AnonymousWP committed:
pkg: add `org.mipay.android.manager`

AnonymousWP committed:
pkg: change description of `com.miui.audiomonitor`

Fixes #630.

AnonymousWP committed:
pkg: change description and removal of `com.miui.core`

Fixes #632

AnonymousWP committed:
pkg: change description of `com.amazon.appmanager`

Fixes #631

AnonymousWP committed:
pkg: add note to `com.miui.core` package

AnonymousWP committed:
readme: add Honor to manufacturer list

AnonymousWP committed:
Add SoftWinner packages

AnonymousWP committed:
pkg: change descriptions of MIUI-apps regarding bootloop

This commit also changes the `removal` from `unsafe` to `expert`

AnonymousWP committed:
pkg: change description of `com.sec.android.diagmonagent`

AnonymousWP committed:
pkg: change description of `com.android.statementservice`

AnonymousWP committed:
pkg(`com.huawei.android.thememanager`): change description

AnonymousWP committed:
pkg: add `com.motorola.blur.service.blur`

AnonymousWP committed:
pkg: add AT&T carrier packages

`com.directv.promo.shade` (Directv Remote) and `com.americanexpress.plenti` (Plenti)

AnonymousWP committed:
pkg: add `com.dti.tim`

AnonymousWP committed:
pkg(`com.dti.tim`): improve description

AnonymousWP committed:
pkg: improve description

AnonymousWP committed:
pkg(`com.coloros.appmanager`): add description

AnonymousWP committed:
pkg(`com.coloros.appmanager`): improve description

AnonymousWP committed:
pkg(`com.motorola.blur.service.blur`): change `removal` to `Advanced`

AnonymousWP committed:
pkg: add Docomo apps

AnonymousWP committed:
pkg(`com.sec.android.app.apex`): change description and removal

AnonymousWP committed:
pkg(`com.lenovo.ue.device`): change description and removal

AnonymousWP committed:
pkg(`com.samsung.android.allshare.service.mediashare`): change description

AnonymousWP committed:
pkg(`com.lenovo.ue.device`): add note to the description

Co-authored-by: Daniel Rudolf <github.com@daniel-rudolf.de>

AnonymousWP committed:
pkg(`com.samsung.android.app.sharelive`): add list, description and removal

AnonymousWP committed:
pkg(`com.samsung.android.app.sharelive`): change removal to `Recommended`

AnonymousWP committed:
pkg: add `com.motorola.gamemode`

AnonymousWP committed:
pkg(`com.motorola.gamemode`): improve description

AnonymousWP committed:
pkg(`com.android.chrome`): add note to description

AnonymousWP committed:
pkg(`com.android.chrome`): change removal to advanced

AnonymousWP committed:
pkg: add and change HeyTap packages

AnonymousWP committed:
pkg: add `com.debug.loggerui`

AnonymousWP committed:
pkg(`com.samsung.SMT`): change description and removal

AnonymousWP committed:
pkg(`com.android.storagemanager`): change removal

AnonymousWP committed:
pkg(`com.samsung.android.mdx.quickboard`): change description and removal

AnonymousWP committed:
pkg(`com.samsung.android.net.wifi.wifiguider`): change description

AnonymousWP committed:
pkg(`com.samsung.android.weather`): change description

AnonymousWP committed:
pkg(`com.sec.android.app.setupwizard`): change description

AnonymousWP committed:
pkg(samsung-cmfa): add `com.samsung.cmfa.AuthTouch`

This commit also changes the `list`, `description` and `removal` for `com.samsung.android.cmfa.framework`

AnonymousWP committed:
pkg(`com.realme.securitycheck`): change description

AnonymousWP committed:
pkg: add `com.google.android.apps.mediahome.launcher`

AnonymousWP committed:
pkg(`com.samsung.android.sm.policy`): improve description

AnonymousWP committed:
pkg(`com.samsung.android.mdx.quickboard`): revert removal

AnonymousWP committed:
pkg(`com.samsung.android.scs`): change removal

AnonymousWP committed:
pkg(`com.samsung.android.game.gos`): improve description

AnonymousWP committed:
pkg(`com.samsung.android.svcagent`): improve description

AnonymousWP committed:
pkg(`com.sec.android.app.sysscope`): change description

AnonymousWP committed:
pkg: add Kyocera Hydro packages

AnonymousWP committed:
pkg: add Samsung Galaxy J2 Prime packages

AnonymousWP committed:
pkg(`com.samsung.ims.smk`): add list and description

AnonymousWP committed:
pkg(miui): add missing packages

AnonymousWP committed:
pkg: change description of MIUI 14 packages

AnonymousWP committed:
pkg: add Redmi 8T packages

AnonymousWP committed:
pkg: add Xiaomi Pad 5 Pro packages

AnonymousWP committed:
pkg(`com.google.android.packageinstaller`): change description

AnonymousWP committed:
docs(readme): fix typo and list

AnonymousWP committed:
pkg(`com.samsung.android.sm.policy`): change description

AnonymousWP committed:
pkg(`com.samsung.app.newtrim`): change description

AnonymousWP committed:
pkg(`com.samsung.android.hdmapp`): add missing list, description and change removal

AnonymousWP committed:
pkg(`com.samsung.android.sm.policy`): fix description

AnonymousWP committed:
pkg(`com.samsung.android.sm.policy`): change description

AnonymousWP committed:
pkg: package documentation for Infinix

AnonymousWP committed:
pkg(`com.bbk.theme`): fix incorrect package name

Commit:
Fixed typos in uad_lists.json

Commit:
Add wiko packages

AnonymousWP commented:
You need to use the closing words in your description, or else the issue won't be closed upon merging, like:

> closes #856.

jvirgovic commented:
Thanks, fixed it just now.

Commit:
com.android.companiondevicemanager edited

Commit:
pkg: com.telus.myaccount

dependabot[bot] committed:
build(deps): bump webpki from 0.22.0 to 0.22.4

Bumps [webpki](https://github.com/briansmith/webpki) from 0.22.0 to 0.22.4.
- [Commits](https://github.com/briansmith/webpki/commits)

---
updated-dependencies:
- dependency-name: webpki
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Frigyes06 committed:
Merge pull request #1 from Frigyes06/dependabot/cargo/webpki-0.22.4

build(deps): bump webpki from 0.22.0 to 0.22.4

Frigyes06 committed:
Updated lockfile, fixes atty issue

Frigyes06 committed:
Downgrade and error fixes

Frigyes06 committed:
Merge pull request #2 from Frigyes06/lockfile-fix

Updated lockfile

Frigyes06 committed:
Merge branch '872-incorrect-name-app-to-vivo-phone' of https://github.com/AnonymousWP/universal-android-debloater

Frigyes06 committed:
Merge branch '869-Infinix-change-recommendation' of https://github.com/AnonymousWP/universal-android-debloater

Frigyes06 committed:
Merge branch '533-534-535-Samsung-packages' of https://github.com/AnonymousWP/universal-android-debloater

Frigyes06 committed:
Merge branch '862-HdmApp' of https://github.com/AnonymousWP/universal-android-debloater into HdmApp

Frigyes06 committed:
Merge branch '861-S-Video-Trim-is-worse-than-useless' of https://github.com/AnonymousWP/universal-android-debloater into S-video-trim-recommendations

Frigyes06 committed:
Merge branch '857-Fix-instructions-list-format' of https://github.com/AnonymousWP/universal-android-debloater into fix-instruction-list-format

Frigyes06 committed:
cleaned up sentences and spelling

Frigyes06 committed:
Merge pull request #5 from Frigyes06/samsung-packages

Samsung packages update

Frigyes06 committed:
Merge branch '850-Package-Installer' of https://github.com/AnonymousWP/universal-android-debloater

Frigyes06 committed:
Merge pull request #4 from Frigyes06/infinix-recommendations

Infinix recommendations

Frigyes06 committed:
Merge branch '838-unlisted-miui-14-+-fixes-documentation' of https://github.com/AnonymousWP/universal-android-debloater into miui-documentation

Frigyes06 committed:
Merge branch '839-S-SMK-display-name' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '810-Samsung-Galaxy-J2-Prime-unlisteds' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '809-Kyocera-Hydro-unlisteds' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '827-SysScope-isn't-Verizon-specific' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '819-SVC-Agent-more-info' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '817-S.G.O.S.-has-a-Game-Intent-Service-' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '530-Samsung-Core-Services-has-a-description-on-Galaxy-Store' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '782-Samsung-Media-&-Devices-is-a-bit-more-useless' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '796-Entertainment-Space-(useless)' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '794-com.realme.securitycheck' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '787-Samsung-CMFA' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '788-Samsung-SetupWizard-notification-if-non-Verizon-SIM' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '785-com.sec.android.daemonapp-is-also-the-Weather-app' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '784-Samsung-Wi-Fi-Tips-description' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '781-Storage-Manager-can-be-disabled-on-my-system' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '779-Samsung-TTS-privilege-escalation-vulnerability' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '776-add-DebugLoggerUI' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '768-Some-apps-that-comes-with-OPPO-phones' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '757-Google-Chrome-Now-Required-Dependency-for-Passkeys' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '742-Add-com.motorola.gamemode-to-the-list' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '736-Quick-Share-App' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '713-Removal-of-com.lenovo.ue.device-causes-system-reset' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '725-AllShare/SmartView-is-also-named-Nearby-Service-' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '718-Remove-com.sec.android.app.apex-from-recommended-list,-its-needed-to-take-motion-pictures-and-to-have-face-recognition-in-the-gallery' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '700-add-Docomo-carrier-packages' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '675-motorola-Blur-data-collector' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '695-AppManager' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '684-com.dti.tim' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '678-AT&T-Carrier-packages-(com.directv.promo.shade,-com.americanexpress.plenti)' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '672-update-description-of-com.huawei.android.thememanager' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '669-com.android.statementservice-update' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '664-Disabling-DiagMonAgent-causes-device-overheating-due-to-crashes' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '641-Non-bootloop-apps' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '643-Unlisted-packs-from-Azpen-A727-(tablet)' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch 'add-honor-as-manufacturer-in-README' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '560-Xiaomi-unlisted-packages' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '631-Mobile-Device-Information-Provider' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '520-Honor-unlisted-packages' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '617-Yandex-bloatware' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '580-LDPlayer-unlisted-apps' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '584-Sony-package-addition' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '612-Alt-name-of-com.samsung.android.app.omcagent' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '610-KPEcore' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '605-Battery-and-Performance' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '604-MIUI-Security-app' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
kept warning

Frigyes06 committed:
Merge branch '586-Fairphone-unlisted-apps-addition' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '593-Undocumented-side-effects-of-removing-some-biometric-related-packages-on-Samsung-Galaxy-S9+-' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '495-add-fonts' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '583-com.android.basicsmsreceiver' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '582-Gallery3d' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '581-So-called-Mi-browser' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '578-com.google.android.googlequicksearchbox-does-more-than-just-the-search-box' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '573-Update-Android-System-WebView-description-to-make-clear-the-effect-of-not-having-a-webview-package-installed' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '574-bookmark-provider' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '563-borneo-Gametime,-Interactive-Wallpaper,-and-Power-Touch-not-removed' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '558-pkg-add-com.sec.app.samsungprintservice' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '552-Samsung-Security-Log-Agent-is-also-responsible-for-permission-monitor-' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '551-update-Samsung-Auto-Doodle-docs' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '549-Add-com.huawei.himovie' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '539-Disabling-stk-or-stk2-removes-home-icon' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '524-Samsung-App-Separation-official-docs' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '512-Altice-Music' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '507,508,509-Huawei-packages-' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '505-Update-Infinix/Tecno-list' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '471-realme-packages' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch 'patch-1' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge branch '477-com.huawei.hwid-not-shows-info-about-potential-break-non-huawei-apps' of https://github.com/AnonymousWP/universal-android-debloater into anonymousWP-patches

Frigyes06 committed:
Merge pull request #6 from Frigyes06/HdmApp

Hdm app

Frigyes06 committed:
Merge pull request #7 from Frigyes06/S-video-trim-recommendations

S video trim recommendations

Frigyes06 committed:
Merge pull request #8 from Frigyes06/fix-instruction-list-format

Fix instruction list format

Frigyes06 committed:
Merge pull request #9 from Frigyes06/Package-installer

Package installer

Frigyes06 committed:
Merge pull request #10 from Frigyes06/miui-documentation

Miui documentation

Frigyes06 committed:
Merge branch 'main' into anonymousWP-patches

Frigyes06 committed:
Merge pull request #11 from Frigyes06/anonymousWP-patches

AnonymousWP all remaining patches

Frigyes06 committed:
Bumped version, modified update url

Frigyes06 committed:
release notes

Frigyes06 committed:
clarified release notes

Frigyes06 committed:
reverted edition change

Frigyes06 committed:
Updated in-app links

Frigyes06 committed:
Merge pull request #12 from Frigyes06/0.5.2-release-prep

0.5.2 release prep

deepsource-io[bot] committed:
ci: add .deepsource.toml

Frigyes06 committed:
Merge pull request #13 from jxd1337/telus-pkg

pkg: com.telus.myaccount

Frigyes06 committed:
Merge branch 'main' into fix-typos

Frigyes06 committed:
Update CHANGELOG.md to include 0.5.2 release

Commit:
Update Wiko support in README

Frigyes06 committed:
Merge pull request #14 from jxd1337/fix-typos

Fixed typos in uad_lists.json

Frigyes06 committed:
Updated changelog

Frigyes06 committed:
fixed links in changelog

Frigyes06 committed:
Merge pull request #16 from jxd1337/companion-device-manager

com.android.companiondevicemanager edited

Frigyes06 committed:
Merge branch 'main' into wiko-packages

Commit:
Update uad_lists.json
